### ISSUE TITLE
Is this possible to setup with 1 PS EYE camera?

### ISSUE THREAD

**Issue by dkuul** [cboulay/PSMoveService#495]
Hi! is this possible to setup with 1 ps eye camera? I'm asking because my movement is inverted or something like that in steam VR, idk maybe I'm just dumb and it's very obvious xD

**Comment by samr28** [comment]
I don't think that it is possible to do with only one camera. Even if it is, it will be super wonky. It can only track in one axis so if it's in front of you, it won't be able to properly detect forward and backwards motion. (Yes, I know that it can track the size of the controller but that doesn't work as well as a second camera.) You should get a second camera :)

**Comment by TrickyAssassin** [comment]
Yess it will work but it won't be a plesant experience
